daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -140,10 +149,11 @@ instance IsParties Party where
 instance IsParties [Party] where
   toParties ps = ps
 
--- | HIDE
 class Template c => TemplateKey c k | c -> k where
+    -- | The key of a contract.
     key : c -> k
 
+    -- | The maintainers of the contract key.
```

Comment:
Also a bit light. Could you add something about what it means to be a maintainer? eg the party/parties guaranteeing that the key is unique.

Also, I worry about saying 'maintainers' by default - implies there are usually more than one, and I'm not sure that's going to be the case.